result/arrow-parens.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-env es6*/
+
+// Bad
+a => {}
+
+// Good
+(a) => {}
+/*eslint-env es6*/
+
+// Bad
+if (a => 2) {
+}
+
+// Good
+if (a >= 2) {
+}
+/*eslint-env es6*/
+
+// Bad
+(a) => {}
+
+// Good
+a => {}
+/*eslint arrow-parens: ["error", "always"]*/
+/*eslint-env es6*/
+
+a => {};
+a => a;
+a => {'\n'};
+a.then(foo => {});
+a.then(foo => a);
```

Comment:
__[development][eslint]__
* Error - 'foo' is defined but never used (no-unused-vars)